### DISCUSSION TITLE
How to MacVlan

### DISCUSSION THREAD

**Discussion by storm1er** [Ysurac/openmptcprouter#2009] [Q&A]
Hi,

I have a pretty basic config:
```
WWAN____WIRELESS___
                    \
WAN1____ETH/USB3____RPI4___
                    /      \
WAN2____ETH/USB3___/        \
                             ETH/ETH
                              \
                               LAN
```

As I use only 4G connections, I would like to limit some LAN devices.
I don't want to use quota, as it would make those devices unusable after a short period of time which is not ideal.

I heard we could do macvlan, but I don't know (at all) how it works.
Could someone explain to me the pre-requisite and how to apply it on my network? 
This way I could use the integrated SQM/QoS to limit the bandwidth of those devices :)

And a big shoutout to @Ysurac and contributors for this project, channel bonding DIY has never been so easy to setup =D

**Comment by storm1er** [comment]
Oh, and btw, I used to use nft-qos on my old mwan3 openwrt, but it clearly does not work with OMR :/ 

It would have been the easiest solution :(

**Comment by Ysurac** [comment]
MacVLAN is the default configuration on RPI, so you can connect all to a switch and use the only available ethernet port of RPI. MacVLAN interfaces can be set via the wizard (System->OpenMPTCProuter) or via Network->Interfaces.

**Reply by storm1er** [reply]
Thanks for the response, but I'm very new to this macvlan stuff and I wasn't able to set up this for now.
Do you have any tutorial or step-by-step stuff I could use?

- Remove the current static eth0@lan
- Create macvlan eth0@vlan
- ?
